src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/ReliableConnectionHelper.cs

```diff
@@ -676,6 +676,12 @@ public class ServerHostInfo
             public string ServicePackLevel;
         }
 
+        public class ServerSysInfo
```

Comment:
nit: unless you're copying the naming from somewhere else, I typically prefer to spell out the whole word (as recommended in .Net naming conventions guide).  So `ServerSystemInfo` in this case.